packages/features/users/components/TwoFactorAuthRequiredBanner.tsx

```diff
@@ -0,0 +1,30 @@
+import { APP_NAME } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import useMeQuery from "@calcom/trpc/react/hooks/useMeQuery";
+import { TopBanner } from "@calcom/ui";
+import { Key } from "@calcom/ui/components/icon";
+
+function TwoFactorAuthRequiredBanner() {
```

Comment:
Banner to prompt users that they are required to setup mfa.
This helps avoid confusion since they will not be able to navigate
to other pages until they setup 2fa.